jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/utils/PayloadCreator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.jparest.utils;
```

Comment:
nit: There is already a dir for test utils, might be better to place in there instead of a new one:
`uk.gov.homeoffice.digital.sas.jparest.testutils`

Reply:
Done, moved the PayloadCreator to the existing util package